packages/website/src/components/typeDetails/TypesDetails.tsx

```diff
@@ -48,15 +48,13 @@ export function TypesDetails({
           value={value}
         />
       </div>
-      {selectedNode && (
```

Comment:
`selectedNode` may be undefined

Reply:
@armano2 based on the types it can't be.
the initial value passed to `useState` is typed as `readonly value: Node`.
Then the only call to `setSelectedNode` is guarded by `if (item.node) { setSelectedNode(item.node) }`